src/DetailsView/components/cards-view.tsx

```diff
@@ -11,6 +13,7 @@ export interface CardsViewProps {
     deps: CardsViewDeps;
     ruleResultsByStatus: UnifiedStatusResults;
     userConfigurationStoreData: UserConfigurationStoreData;
+    targetAppInfo: TargetAppData;
```

Comment:
What do we think about this approach vs including `TargetAppData` in `UnifiedStatusResults/UnifiedRuleResult`? If the `UnifiedStatusResults` is meant to map to card ViewModels, it might make sense because each card needs the target app data; some of the data might be duplicated.

Reply:
Yeah, my thought was:
First we have store data, where duplication should be avoided thus target app data is a separate field. Then, we have the view model, in which data like target app data, rule info, env info, will all be duplicated in each failed instance data. The question is where should this conversion/mapping happen. My inclination is that the mapping happens as close to the consumer as possible, but feel free to let me know if there are good reasons to do it earlier.